### PR TITLE
Governing Body edit page

### PR DESCRIPTION
**Pull request by anitacaron**

### CODE REVIEW

File: app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <:action>
         {{! TODO: Replace this with a link once the "edit" route is created }}
```
Comment (Windvis) [inline app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs:10]: Don't forget to remove this :smile:

**Review by Windvis — CHANGES_REQUESTED** [review]
:rocket:

File: app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs (hunk 1 above)
Comment (anitacaron) [reply app/templates/administrative-units/administrative-unit/governing-bodies/governing-body/index.hbs:10]: 🙄

**Review by Windvis — APPROVED** [review]

**Comment by Windvis** [comment]
Woops, it seems there was a second edit link in the overview page table :grimacing:

**Comment by anitacaron** [comment]
It's for the mandatory, no?

**Pull request merged**